test/tstGeometryIntersects.cpp

```diff
@@ -41,7 +40,7 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(intersects_point_box, Coordinate, CoordinatesList)
 
 BOOST_AUTO_TEST_CASE_TEMPLATE(intersects_box_box, Coordinate, CoordinatesList)
 {
-  using ArborX::Details::intersects;
+  using ArborX::intersects;
```

Comment:
Did you mean to keep that using-directive?
It looks like we are inconsistent throughout that file.

Reply:
Yes, I kept it. We are consistent within file with respect to `ArborX::intersects`. I am inconsistent sometimes with respect to other things. I don't think it matters much inside the tests.